blueprints/ember-segment/index.js

```diff
@@ -1,37 +0,0 @@
-'use strict';
-
-module.exports = {
-  normalizeEntityName: function() { },
-
-  beforeInstall: function() {
-    var eventName = 'gatherPageAnalytics';
-
-    var reopens = [
```

Comment:
because i don't think it should be automatically included, these things are usually very custom, the event name, the route naming (you may want to rewrite the name with a different name depending on the business needs) etc.